pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -386,10 +359,6 @@ func (v *vpnSeedServer) podTemplate(configMap *corev1.ConfigMap, dhSecret, secre
 							Name:      volumeNameTLSAuth,
 							MountPath: volumeMountPathTLSAuth,
 						},
-						{
-							Name:      volumeNameDH,
-							MountPath: volumeMountPathDH,
```

Comment:
Nit: You could also drop https://github.com/gardener/gardener/blob/be23065e1441a703eef7f2ed565c865ab9f43017/pkg/component/networking/vpn/seedserver/seedserver.go#L92 and https://github.com/gardener/gardener/blob/be23065e1441a703eef7f2ed565c865ab9f43017/pkg/component/networking/vpn/seedserver/seedserver.go#L85.